spec/index.bs

```diff
@@ -10050,9 +10050,6 @@ dictionary GPUComputePassDescriptor : GPUObjectDescriptorBase {
                         - |pipeline| is [$valid to use with$] |this|.
                     </div>
                 1. Set |this|.{{GPUComputePassEncoder/[[pipeline]]}} to be |pipeline|.
-
-                1. [$Enqueue a compute command$] on |this| which issues the subsequent steps on the
-                    [=Queue timeline=].
```

Comment:
Removed this because there were no steps to execute on the queue timeline after your changes.